src/globus_cli/commands/endpoint/create.py

```diff
@@ -84,7 +83,10 @@ def endpoint_create(
     **kwargs: Any,
 ) -> None:
     """
-    Create a new endpoint.
+    Create a new endpoint. (deprecated)
+
+    This command is deprecated. Either `globus gcp create` or the Globus Connect Server
+    CLI should be used instead for most cases.
```

Comment:
Do we also want to add a deprecation warning to the command's output, or would that be too noisy?

Reply:
Not _yet_, but we will in the future. I discussed this with Brigitte, and she agreed with my decision to use the current (subtle) doc note.